scene/main/node.cpp

```diff
@@ -3480,9 +3480,19 @@ static void _print_orphan_nodes_routine(Object *p_obj) {
 		path = String(p->get_name()) + "/" + p->get_path_to(n);
 	}
 
+	String source;
+	Variant script = n->get_script();
+	if (!script.is_null()) {
+		Resource *obj = Object::cast_to<Resource>(script);
+		source = obj->get_path();
+	} else {
+		source = "";
+	}
```

Comment:
`String` is implicitly empty on init, so this is redundant
```suggestion
	}
```

Reply:
With every comment on this PR I have learned a new thing, thank you! This change has been made.